dev-lang/haxe/haxe-4.3.6-r2.ebuild

```diff
@@ -28,7 +28,7 @@ RESTRICT="strip"
 RDEPEND="
 	<=dev-ml/extlib-1.7.9:=
 	>=dev-lang/ocaml-4:=[ocamlopt?]
-	~dev-ml/luv-0.5.12:=
+	~dev-ml/luv-0.5.14:=
```

Comment:
It's not a minimum version here though, it pins it to 0.5.14-r0/r1/r2 and so on. Should the dependency be looser instead?

Reply:
I think it's fine as it is.

Reply:
But it isn't, because you hit this bug to begin with, and it was pinning you to .12. Now we're just pinning to .14, when clearly on non-musl, that dep wasn't even accurate (because .14 worked fine).

Reply:
@xgqt The upstream dep in `haxe.opam` is `"luv" {>= "0.5.13"}`, so I think `>=0.5.13:=` is fine here, yeah?

Reply:
No, I still think the change is wrong, I'm afraid. It's just continuing the problem. If the dep hadn't been pinned to begin with, you wouldn't have had the issue you did on musl.

There's no need for it to be _pinned_. It should have a minimum version of 0.5.13, not "exactly 0.5.14, modulo revision".

Reply:
Oh I didn't notice that.
I remplaced the pinned version with the proper minimal version requirement as I actually wanted to do.